net/xray-core/Makefile

```diff
@@ -30,6 +30,7 @@ define Package/xray/template
   TITLE:=A platform for building proxies to bypass network restrictions
   SECTION:=net
   CATEGORY:=Network
+  SUBMENU:=Web Servers/Proxies
```

Comment:
The same as https://github.com/openwrt/packages/pull/25391#discussion_r1845437256.